doc/dev.html

```diff
@@ -535,17 +526,9 @@ <h4 id="system-tests">System Tests</h4>
   At a minimum, a system test always attempts to compile generated tests.
   It should also try to run any generated tests and confirm that regression tests
   pass, and that error tests fail.
-  Looking at the <code>RandoopSystemTest.generateAndTest()</code> method may be
-  helpful in understanding what can be done.
-</p>
-<p>
-  The output of the Randoop run is captured in the <code>RandoopRunDescription</code>
-  object returned by <code>generateAndCompile</code>, and the output from running
-  the tests is captured in the <code>TestRunDescription</code> returned by <code>runTests</code>.
-  This output can be used in other assertions as needed.
+  Additional information can be found in the <code>RandoopSystemTest</code> class.
 </p>
 
-
```

Comment:
I personally like to put a larger break before a section header (or at other places in a file where the topic changes).